rfc9114.md

```diff
@@ -1615,7 +1615,7 @@ PUSH_PROMISE Frame {
 The payload consists of:
 
 Push ID:
-: A variable-length integer that identifies the server push operation.  A Push
+: A variable-length integer that identifies the server push operation.  A push
   ID is used in push stream headers ({{server-push}}) and CANCEL_PUSH frames
   ({{frame-cancel-push}}).
 
```

Comment:
On lines 1626-1645, I think this needs to be lowercase: "A server MUST NOT use a Push ID that is larger than" (six instances)